.github/actions/install-wdk/action.yml

```diff
@@ -131,13 +131,75 @@ runs:
         } else {
 
           Write-Host "Using Winget to install WDK version $inputVersion..."
-          $wdk = "Microsoft.WindowsWDK.$inputVersion"
-          if ((Get-WinGetPackage -Id $wdk -Source winget -MatchOption Equals).Id -eq '$wdk') {
-            Write-Host "$wdk is already installed. Attempting to update..."
-            Update-WinGetPackage -Id $wdk -Source winget -MatchOption Equals -Mode Silent -Force
-          } else {
-            Write-Host "Installing $wdk..."
-            Install-WinGetPackage -Id $wdk -Source winget -MatchOption Equals -Mode Silent -Force
+          
+          # Import the WinGet PowerShell module  
+          Write-Host "Importing WinGet PowerShell module..."
+          Import-Module Microsoft.WinGet.Client -Force
+          
+          # Detect and uninstall any existing WDK packages to avoid header conflicts
+          Write-Host "Detecting and removing existing WDK packages..."
+          try {
+            $existingWDKs = Get-WinGetPackage -Source winget | Where-Object { $_.Id -like "Microsoft.WindowsWDK*" }
```

Comment:
Similar to the feedback about SDK, this can sometimes have a name that contains "Windows Driver Kit"